### DISCUSSION TITLE
Dead project?

### DISCUSSION THREAD

**Discussion by OptimoSupreme** [lxdware/lxd-dashboard#90] [General] [answered]
Hello! I love having an open source alternative to Canonical's web interface, and I actually think this one is generally better. However I'm wondering if this project is still actively being maintained and developed?

**Comment by matthewalanpenning** [comment, marked as answer]
I ported this project to https://github.com/PenningLabs/lxconsole to move from PHP to Python as a based code and make improvements to the way the software works. I am updating this project if there is a vulnerability, but my focus is on the lxconsole version which works with both LXD and Incus.
